spring-boot-starters/spring-boot-starter-data-geode/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starters</artifactId>
+		<version>1.4.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-boot-starter-data-geode</artifactId>
+	<name>Spring Boot Data Geode Starter</name>
+	<description>
+		The Spring Boot Data Geode Starter including Spring Data GemFire with support for Apache Geode.
+	</description>
+	<url>http://projects.spring.io/spring-boot/</url>
+	<organization>
+		<name>Pivotal Software, Inc.</name>
+		<url>http://www.spring.io</url>
+	</organization>
+	<properties>
+		<main.basedir>${basedir}/../..</main.basedir>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.data</groupId>
+			<artifactId>spring-data-gemfire</artifactId>
+			<version>${spring-data-geode.version}</version>
```

Comment:
> You could also override the dependency for a particular module by specifying a dependency management for it

Right, good point!

I could see a user wanting to do this over overriding the entire SD release train, presumably because the user wants specific (perhaps, upcoming) features in particular SD module, and is either not using the other SD modules, or happy with the SB recommended/vetted version.